internal/testhelpers/testhelpers.go

```diff
@@ -1,3 +1,4 @@
+// Package testhelpers contains functions that pretty-print test failures.
```

Comment:
```suggestion
// Package testhelpers contains functions which pretty-print test failures.
```

Reply:
`that` vs `which` rundowns were not on my expected path for today, but I'll take it.